openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx

```diff
@@ -686,42 +665,6 @@ const ServicePage: FunctionComponent = () => {
     }
   };
 
-  const getDeleteEntityMessage = () => {
-    const service = serviceName?.slice(0, -1);
-
-    switch (serviceName) {
-      case ServiceCategory.DATABASE_SERVICES:
-        return getEntityDeleteMessage(
-          service || 'Service',
-          `${pluralize(instanceCount, 'Database')}, ${pluralize(
-            schemaCount,
-            'Schema'
-          )} and ${pluralize(tableCount, 'Table')}`
-        );
-
-      case ServiceCategory.MESSAGING_SERVICES:
-        return getEntityDeleteMessage(
-          service || 'Service',
```

Comment:
do we need to remove this function? cc @Sachin-chaurasiya i think this is used to show how many database schema is available in database

Reply:
This function is added to utils. Now this message will display when one tries to permanently delete a service.